server/src/main/java/org/candlepin/model/AsyncJobStatus.java

```diff
@@ -101,22 +101,31 @@ public boolean isTerminal() {
     @NotNull
     private String name;
     private String origin;
+    private String executor;
     private String principal;
 
-    // TODO: If we add more stuff like this field, we should create another table to store
-    // job metadata rather than continuing to bolt more columns onto this for fields that may
-    // not be filled/configured globally (i.e. announce completion and job log level)
-    @Column(name = "correlation_id")
-    private String correlationId;
+    @Column(name = "log_level")
+    private String logLevel;
+    @Column(name = "log_execution_details")
+    private boolean logExecutionDetails;
+
+    @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable(name = "cp2_async_job_metadata", joinColumns = @JoinColumn(name = "job_id"))
```

Comment:
Somewhat force of habit, but it's intended to denote tables added in 2.x. As far as the format being unique to per-org data duplication, I think that's a bit of a misunderstanding with the documentation. Can you point me to where that is?

Reply:
I wasn't sure what exactly this prefix denotes, and this is the only thing I found: https://www.candlepinproject.org/docs/candlepin/cp2_migration.html#database-upgrade

Reply:
Ahh, okay.

Anyway, it's a fair point and it may not be something necessary to carry forward (also gets in the way of tab completion in some cases). I'll send out an e-mail for a quick gathering of thoughts on whether or not we should keep the cp2_ prefix for new tables, or start getting everything back to cp_ where possible. I expect most people won't have opinions one way or another, so if the general response is apathy, I'll drop the 2 here.

Reply:
Actually scratch that. I can't think of any reason to keep it, and the tab completion problem, coupled with the extra character wasting our already-limited table name space is probably more than enough reason to drop it. I'll get that fixed up here momentarily.